conf/modules/stereocam_stereocam2state.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="stereocam2state" dir="stereocam/stereocam2state">
+  <doc>
+    <description>This module sends the data retreived from an external stereocamera modules, to the state filter of the drone. This is done so that the guidance modules can use that information for couadcopter</description>
+  </doc>
+  <header>
+    <file name="stereocam2state.h"/>
+  </header>
+  <init fun="stereo_to_state_init()"/>
```

Comment:
If you don't need the init and start stop functions, please remove them and only keep the periodic that you actually use...